test/end-to-end/offer.js

```diff
@@ -332,7 +334,8 @@ describe("Offer endpoint tests", () => {
                 const offer_params = {
                     ...offer,
                     owner: test_company._id,
-                    ownerName: test_company.name
+                    ownerName: test_company.name,
+                    ownerLogo: test_company.logo,
```

Comment:
remove these and assert that they are present in the created offer (after L350), since they should be added automatically

Reply:
Addressed requested changes